docs/topics/signalen-and-standard-for-public-code.md

```diff
@@ -27,55 +27,63 @@ Continuous integration tests SHOULD validate that the source code and the policy
-Code and its documentation SHOULD NOT contain situation-specific information. For example, personal and organizational data as well as tokens and passwords used in the production system should never be included. | partial | The software is easily configurable, the default configuration is Amsterdam-specific.
+Code and its documentation SHOULD NOT contain situation-specific information.  | partial | The software is easily configurable, the default configuration is Amsterdam-specific.
 
-## [Welcome contributions](https://standard.publiccode.net/criteria/open-to-contributions.html)
+## [Welcome contributors](https://standard.publiccode.net/criteria/open-to-contributions.html)
 
-- [ ]
+- [ ] compliant with this criterion.
 
 Requirement | meets | links and notes
 -----|-----|-----
-The codebase MUST have a public issue tracker that accepts suggestions from anyone. | yes | Amsterdam has a second private issue tracker.
 The codebase MUST allow anyone to submit suggestions for changes to the codebase. | yes |
-The documentation MUST link to both the public issue tracker and submitted codebase changes, for example in a README file. | no | The two different issue trackers in the Amsterdam and Signalen namespaces make this a bit challenging. Maybe disable the issue tracker on the Amsterdam repo and link to the Signalen one?
-The codebase MUST include an email address for security issues and responsible disclosure. | no | Consider a SECURITY.md file.
 The codebase MUST include contribution guidelines explaining how contributors can get involved, for example in a CONTRIBUTING file. | no | The docs/CONTRIBUTING.md still says "we do not accept merge requests".
-The project MUST have communication channels for users and developers, for example email lists. | (maybe) | Add link to Slack on signalen.org and [email list](https://lists.publiccode.net/mailman/postorius/lists/signalen-discuss.lists.publiccode.net/)?
-The codebase SHOULD have a publicly available roadmap. | no | There is a [product steering group](https://github.com/orgs/Signalen/projects/2), but no roadmap yet.
 The codebase SHOULD advertise the committed engagement of involved organizations in the development and maintenance. | (maybe) | The signalen.org website shows the logos.
-The documentation SHOULD include instructions for how to report potentially security sensitive issues on a closed channel. | no | Document the process in SECURITY.md?
 The codebase SHOULD document the governance of the codebase, contributions and its community, for example in a GOVERNANCE file. | no |
+The codebase SHOULD have a publicly available roadmap. | no | There is a [product steering group](https://github.com/orgs/Signalen/projects/2), but no roadmap yet.
 The codebase MAY include a code of conduct for contributors. |  |
 
+The documentation SHOULD include instructions for how to report potentially security sensitive issues on a closed channel. | no | Document the process in SECURITY.md?
```

Comment:
This one was moved down (but the comments was lost on the way)

Reply:
Good Catch!